City-of-Helsinki/drupal-helfi-platform-config
# UHF-X Remove force enabled ru

Test with no previous setup:

1. Run the composer one liner to get the site: `composer create-project City-of-Helsinki/drupal-helfi-platform:dev-main hel-platform --no-interaction --repository https://repository.drupal.hel.ninja/`
2. Go to the newly created folder `hel-platform` and switch to correct branch: `composer require drupal/helfi_platform_config:dev-UHF-X_remove_force_enabled_ru`
3. Get the site running: `make new`
4. Enable the example content module `make drush-enable-example-content`
5. Log in to the site using the link that the install gives and check that the site works as before, but without ru-language enabled. F.e. Go to https://hel-platform.docker.so/en/node/add/page and check that there is no "Russian" in the language option.

Test with helfi, kymp or sote:
1. Switch to correct branch: `composer require drupal/helfi_platform_config:dev-UHF-X_remove_force_enabled_ru`
2. Get the site running: `make new`
3. Log in to the site using the link that the install gives and check that the site works as before. Check it is still possible to add Russian content by going to https://helfi.docker.so/en/node/add/page and choosing "Russian" from the language option.
